charts/pega/templates/_pega-deployment.tpl

```diff
@@ -77,6 +77,15 @@ spec:
 {{- if .node.nodeSelector }}
       nodeSelector:
 {{ toYaml .node.nodeSelector | indent 8 }}
+{{- end }}
+{{- if (ne .root.Values.global.provider "openshift") }}
+      securityContext:
+        fsGroup: 0
+{{- if .node.securityContext }}
```

Comment:
Should this also check for .node.securityContext.runAsUser presense as well? If in case when someone adds securityContext.fsGroup as parameter or another parameter this might force the user to always pass .node.securityContext.runAsUser or else nil pointer would come in next line.

Reply:
Verified that our deployments works only with root user as fsGroup. Hence not providing it as an overridable option. And I tried to add nil check for runAsUser `{{- if and (.node.securityContext) (.node.securityContext.runAsUser) }}` but looks like it doesn't work as it was unable to find child node with parent node also commented in values.yaml